opm/material/fluidsystems/Spe5FluidSystem.hpp

```diff
@@ -274,7 +274,7 @@ class Spe5
             ? 0.6500
             : (compIdx == C20Idx)
             ? 0.8500
-            : 1e100;
+            : 1e30;
```

Comment:
I agree that it is definitely not the orthodox or even a nice solution, but IMO it is much easier to read than replacing `1e30` by

```c++
std::numeric_limits<Scalar>::max()
```